.scalafmt.conf

```diff
@@ -1,19 +1,39 @@
 version="3.5.3"
```

Comment:
I'm puzzled that this wasn't done in PR #54 . Could you please check scalastyle-config.xml too?

Reply:
That file was modified as part of the upgrade and looks good.